src/main/java/com/nirima/jenkins/plugins/docker/DockerJobProperty.java

```diff
@@ -60,6 +63,19 @@ public boolean isCleanImages() {
 
     @Extension
     public static final class DescriptorImpl extends JobPropertyDescriptor {
+        public FormValidation doCheckAdditionalTag(@QueryParameter String additionalTag){
+            if (additionalTag.trim().length() == 0){
+                return FormValidation.ok(); // skip zero value
+            }
+            // Exclude 500 error during commit
```

Comment:
This can provide wrong message for entered variables like ${BUILD_NUMBER}.
Will check how this can be fixed...